backend/sozisel/lib/sozisel_web/schema/types/event_types.ex

```diff
@@ -0,0 +1,53 @@
+defmodule SoziselWeb.Schema.Types.EventTypes do
+  use SoziselWeb.Schema.Notation
+
+  object :event do
+    field :id, non_null(:id)
+    field :name, non_null(:string)
+    field :start_minute, non_null(:integer)
+    field :event_type, non_null(:quiz)
```

Comment:
When I think about it `event_type` is an invalid name for something that is more than just a name indicating the type. We will have whole structures here. Also this field should be a union of our custom types, `quiz` for now but other types in the future. Read about graphql unions and how to use them in absinthe